### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by pablo-rodriguez-jd**

### CODE REVIEW

File: generators/app/templates/src/components/AuthNavBar/auth-nav-bar.container.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+import { withTranslation } from "react-i18next";
 import AuthNavBar from "./auth-nav-bar.component";
 import data from "@solid/query-ldflex";
 
 // hasPhoto context
 const hasPhotoContext = "http://www.w3.org/2006/vcard/ns#hasPhoto";
-// img context 
-const imgContext = "http://xmlns.com/foaf/0.1/img"
+// img context
+const imgContext = "http://xmlns.com/foaf/0.1/img";
```
Comment (jairo-campos-JD) [inline generators/app/templates/src/components/AuthNavBar/auth-nav-bar.container.js]: are you using imageConext ? if you are using we can access by data.user.foaf_image

File: generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+  ];
   return (
     <NavBar
       navigation={navigation}
       toolbar={[
+        {
+          component: () => (
+            <LanguageDropdown {...props} onLanguageSelect={onLanguageSelect} />
+          ),
+          label: "Language",
```
Comment (james-martin-jd) [inline generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js]: Should this label be derived from the translation service?

File: generators/app/templates/src/containers/Welcome/welcome.component.js [hunk 3]
```diff
@@ -41,174 +40,124 @@ const WelcomePageContent = props => {
...
             />
           </ImageWrapper>
           <p>
-            All Done ? <LogoutButton />
+            {t("welcome.doneMessage")}{" "}
+            <LogoutButton>{t("navBar.logOut")}</LogoutButton>
           </p>
         </WelcomeProfile>
       </WelcomeCard>
       <WelcomeCard className="card">
```
Comment (james-martin-jd) [inline generators/app/templates/src/containers/Welcome/welcome.component.js]: I believe this second WelcomeCard is a merge issue

File: generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js (hunk 2 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js]: Solved, I removed those labels since they're not being used.

**Pull request merged**